### PR TITLE
refactor(presentation): extra hint upload limit

### PR DESCRIPTION
**Pull request by Arthurk12**
### What does this PR do?

Adds a text hint about the accepted presentation file size limit on the presentation upload modal.

presentation_hint

### Motivation

Currently, the user is informed about the presentation size limit only when a uploading error occurs.
Therefore, this is intended to inform users about the presentation size limit before they attempt to upload a file.

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -600,6 +600,8 @@ public:
     restoreOnUpdate: false
     oldMinimizeButton: false
     uploadEndpoint: '/bigbluebutton/presentation/upload'
+    uploadSizeMax: 30000000
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml:603]: This is [partially] reverting https://github.com/bigbluebutton/bigbluebutton/pull/12100 where the agreement was to rely on the single configuration - bigbluebutton.properties
https://github.com/bigbluebutton/bigbluebutton/blob/v2.4.x-release/bigbluebutton-web/grails-app/conf/bigbluebutton.properties#L69-L75

Comment (antobinary) [reply bigbluebutton-html5/private/config/settings.yml:603]: Could we reword it as [something like]:

```
# mirroredFromBBBCore are values controlled in bbb-web properties file. We include a copy here for notification purposes
mirroredFromBBBCore: 
  - uploadSizeMax: 30000000
  - uploadPagesMax: 200
```

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx [hunk 2]
```diff
@@ -1038,6 +1063,7 @@ class PresentationUploader extends Component {
 
           <div className={styles.modalHint}>
             {`${intl.formatMessage(intlMessages.message)}`}
+            {this.renderExtraHint()}
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx:1066]: Could this be optional, controlled in settings.yml ? Something like

`displayHintAboutPresentationConstraints: false`

File: bigbluebutton-html5/private/config/settings.yml (hunk 1 above)
Comment (Arthurk12) [reply bigbluebutton-html5/private/config/settings.yml:603]: Yes, it seems more coherent that way.
I will change it soon.

**Pull request merged**